src/core_plugins/kibana/public/management/app.html

```diff
@@ -14,9 +14,11 @@
 
       <!-- Tabs. -->
       <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
-        <span ng-if="!sectionName" class="kuiLocalTab">
-          {{::section.display}}
-        </span>
+        <header ng-if="!sectionName" id="tabHeader" tabindex="0">
```

Comment:
I agree about the cursor: pointer and hover color here if it's not interactive. However I can go either way on it being tabbable based on the idea that I expressed in the other issue - orienting the user on the page and audibly saying the name of the section.